contracts/cw1-subkeys/helpers.ts

```diff
@@ -346,10 +346,10 @@ const CW1 = (client: SigningCosmWasmClient): CW1Contract => {
   
   const upload = async (): Promise<number> => {
     const meta = {
-      source: "https://github.com/CosmWasm/cosmwasm-plus/tree/v0.1.1/contracts/cw1-subkeys",
+      source: "https://github.com/CosmWasm/cosmwasm-plus/tree/v0.2.1/contracts/cw1-subkeys",
       builder: "cosmwasm/rust-optimizer:0.10.1"
```

Comment:
```suggestion
      builder: " cosmwasm/workspace-optimizer:0.10.3"
```
Should I apply this change?

Reply:
I suppose we change the meta in every contract after we apply this change.